src/libs/actions/App.js

```diff
@@ -135,6 +144,43 @@ function fixAccountAndReloadData() {
         });
 }
 
+/**
+ * Wait for the navigation to be ready, get the initial URL and make sure the
+ * user is logged in. Next, create a new free policy if needed, load policies,
+ * and navigate to the transition exit route if needed.
```

Comment:
NAB, we're leaving a lot of "what" and not a lot of "why" - some general suggestions would be to explain:

- Why we have to wait for the navigation to be "ready"
- Why we are using `getInitialURL()`
- Why we have logic that logs out the current user
- Any other details about `exitTo`
- Any other information someone might need to know that they can't easily tell from looking at this code.

Reply:
Never mind, while explaining `isLoggingInAsNewUser` that logic came up.